when_fetching_events_for_an_event_type/that_has_committed_events.cs

```diff
@@ -45,7 +46,15 @@ public class that_has_committed_events : given.an_event_store
 
         It should_contain_all_the_events_of_the_specified_type = () => 
         {
-            result.Select(e => e.ToEventEnvelope()).ShouldContainOnly(simple_events);
+            var envelopes = result.Select(e => e.ToEventEnvelope());
```

Comment:
This isn't quite the same.  This is checking that there is at least one event in the result for each event, not that there is exactly one event for each result.